backend/ibutsu_server/controllers/result_controller.py

```diff
@@ -98,16 +98,15 @@ def get_result_list(filter_=None, page=1, page_size=25, estimate=False, token_in
     user = User.query.get(user)
     query = Result.query
     if user:
-        query = add_user_filter(query, user)
+        query = add_user_filter(query, user, filter_=filter_, model=Result)
+
     if filter_:
         for filter_string in filter_:
             filter_clause = convert_filter(filter_string, Result)
             if filter_clause is not None:
                 query = query.filter(filter_clause)
 
-    if estimate and not filter_:
```

Comment:
This is partially addressing #238, but there's still some work to be done as the estimated count is still inaccurate.